Cargo.lock

```diff
@@ -1985,18 +1945,18 @@ checksum = "ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"
 
 [[package]]
 name = "openssl-src"
-version = "111.25.3+1.1.1t"
+version = "300.1.5+3.1.3"
```

Comment:
Fyi, this will update openssl from 1.1.1 -> 3.1.3

Reply:
thanks for the heads up! we're gonna test it out in staging